generators/client/templates/angular/src/main/webapp/app/layouts/navbar/_navbar.component.ts

```diff
@@ -21,7 +21,7 @@ import { Router } from '@angular/router';
 import { NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
 import { JhiLanguageService } from 'ng-jhipster';
 
-import { ProfileService } from '../profiles/profile.service'; // FIXME barrel doesn't work here
+import { ProfileService } from '../profiles/profile.service';
```

Comment:
All the places were injecting the service straightaway from the service file rather than common export file.